src/storm/storage/MaximalEndComponentDecomposition.h

```diff
@@ -105,8 +108,8 @@ class MaximalEndComponentDecomposition : public Decomposition<MaximalEndComponen
      * @param choices The choices of the subsystem to decompose.
      */
     void performMaximalEndComponentDecomposition(storm::storage::SparseMatrix<ValueType> const& transitionMatrix,
-                                                 storm::storage::SparseMatrix<ValueType> backwardTransitions, storm::storage::BitVector const* states = nullptr,
-                                                 storm::storage::BitVector const* choices = nullptr);
+                                                 storm::storage::SparseMatrix<ValueType> const& backwardTransitions,
+                                                 storm::storage::BitVector const* states = nullptr, storm::storage::BitVector const* choices = nullptr);
```

Comment:
We hardly use raw pointers, in particular in the API. Is it necessary that these are pointers here? If so, I guess we should document ownership etc

Reply:
This is now using storm::OptionalRef, see #414